src/runtime/pkg/katautils/katatrace/tracing.go

```diff
@@ -130,7 +130,7 @@ func Trace(parent context.Context, logger *logrus.Entry, name string, tags ...ma
 		if logger == nil {
 			logger = kataTraceLogger
 		}
-		logger.WithField("type", "bug").Error("trace called before context set")
+		logger.WithField("type", "bug").WithField("name", name).Error("trace called before context set")
```

Comment:
Note only: two `WithField` or one`WithFields` that accepts a map parameter are OK.